aidial_assistant/utils/state.py

```diff
@@ -41,7 +63,9 @@ def parse_history(history: list[Message]) -> list[ScopedMessage]:
                 messages.append(
                     ScopedMessage(
                         scope=MessageScope.INTERNAL,
-                        message=ModelMessage.assistant(invocation["request"]),
+                        message=ModelMessage.assistant(
+                            _normalize_commands(invocation["request"])
```

Comment:
Why do we need to normalize something here? Why not save the invocation in already normalized form?

Reply:
This is for backward compatibility. Renamed the method.

Reply:
Please elaborate in a doc for `_convert_old_commands` **why** we are doing this: 
1. the format of commands used to be different, 
2. the commands were saved to a state, 
3. state is saved in the chat conversation, 
4. chat conversation is saved in a local storage in chat frontend, 

thus certain users may still have conversations with commands in an old format. So the conversion is done for  
a backward compatibility.

Reply:
Added description.